docs/self-hosted/debian.rst

```diff
@@ -27,25 +27,17 @@ CrateDB repositories:
 
 .. code-block:: sh
 
-   # Add HTTPS support
-   sh$ sudo apt install apt-transport-https
-
-   # Download the CrateDB GPG key
-   sh$ wget https://cdn.crate.io/downloads/apt/DEB-GPG-KEY-crate
-
-   # Add the key to Apt
-   sh$ sudo apt-key add DEB-GPG-KEY-crate
-
-   # Add CrateDB repositories to Apt
-   # `lsb_release -cs` returns the codename of your OS
-   echo "deb https://cdn.crate.io/downloads/apt/stable/ $(lsb_release -cs) main" |
-     sudo tee /etc/apt/sources.list.d/crate-stable.list
+   # Add the CrateDB GPG key
+   sh$ wget -qO- https://cdn.crate.io/downloads/apt/DEB-GPG-KEY-crate | sudo tee /etc/apt/trusted.gpg.d/crate.asc
```

Comment:
apt-key emits deprecation warnings. Output of `man 8 apt-key`:

```
DEPRECATION
       Except for using apt-key del in maintainer scripts, the use of apt-key is deprecated. This section shows how to replace existing use of apt-key.

       If your existing use of apt-key add looks like this:

       wget -qO- https://myrepo.example/myrepo.asc | sudo apt-key add -

       Then you can directly replace this with (though note the recommendation below):

       wget -qO- https://myrepo.example/myrepo.asc | sudo tee /etc/apt/trusted.gpg.d/myrepo.asc

       Make sure to use the "asc" extension for ASCII armored keys and the "gpg" extension for the binary OpenPGP format (also known as "GPG key public ring"). The binary OpenPGP format works for all apt versions, while the
       ASCII armored format works for apt version >= 1.4.

       Recommended: Instead of placing keys into the /etc/apt/trusted.gpg.d directory, you can place them anywhere on your filesystem by using the Signed-By option in your sources.list and pointing to the filename of the key.
       See sources.list(5) for details. Since APT 2.4, /etc/apt/keyrings is provided as the recommended location for keys not managed by packages. When using a deb822-style sources.list, and with apt version >= 2.4, the
       Signed-By option can also be used to include the full ASCII armored keyring directly in the sources.list without an additional file.
```

Reply:
Thanks. We followed that guideline at [^1] already, and removed the use of `apt-key` there.

[^1]: https://cratedb.com/docs/crate/tutorials/en/latest/basic/index.html#configure-package-repository

Reply:
I added a fixup to update the second location too. We should eventually remove the duplication.
(I'd still go with having dedicated pages, it's easier to see what belongs together - having parallel tracks within a single page can get confusing because it's easy to miss where the tracks join again and where you should continue reading.)